soda_machine.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "soda"
```

Comment:
why are you requiring this file? I think this is already done in the spec helper file